.github/workflows/ostests-e2e.yaml

```diff
@@ -113,7 +113,7 @@ jobs:
           jq --version
 
           mkdir -p "$(dirname -- "$TF_VAR_k0sctl_executable_path")"
-          curl --proto '=https' --tlsv1.2 --retry 5 --retry-all-errors -sSLfo "$TF_VAR_k0sctl_executable_path" "https://github.com/k0sproject/k0sctl/releases/download/${K0SCTL_VERSION}/k0sctl-linux-x64"
+          curl --proto '=https' --tlsv1.2 --retry 5 --retry-all-errors -sSLfo "$TF_VAR_k0sctl_executable_path" "https://github.com/k0sproject/k0sctl/releases/download/${K0SCTL_VERSION}/k0sctl-linux-${OSTESTS_ARCH/amd64/x64}"
```

Comment:
Hmm, k0sctl is executed on the runner, and the runner arch is always amd64 (runner arch != cluster arch) ... :thinking:

/cc #4772